boa3/builtin/contract/__init__.py

```diff
@@ -25,3 +25,10 @@ def abort():
     Abort the execution of a smart contract
     """
     pass
+
+
+def exit():
```

Comment:
This should be the implementation of python's built-in `exit`, shouldn't it?
You don't need to include it in the boa3's built-ins